spec/system/monthly_statistics/monthly_statistics_page_spec.rb

```diff
@@ -6,14 +6,74 @@
     FeatureFlag.activate('publish_monthly_statistics')
     create_application_choices
     create_monthly_stats_report
+    visit_monthly_statistics_page
   end
 
-  scenario 'User visits monthly statistics page' do
-    when_i_visit_the_monthly_statistics_page
+  scenario 'User views monthly statistics page' do
```

Comment:
As discussed, we can keep this spec slim and happy — perhaps we could have a request spec or something which checked all the reports worked and they came back with correct `Content-Type` etc?